app/frontend/src/features/messages/surfing/HostRequestView.tsx

```diff
@@ -162,33 +149,29 @@ export default function HostRequestView() {
         <PageTitle className={classes.title}>
           {!title || hostRequestError ? <Skeleton width="100" /> : title}
         </PageTitle>
-
-        <HeaderButton
-          onClick={handleClick}
-          aria-label="Menu"
-          aria-haspopup="true"
-          aria-controls="more-menu"
-          innerRef={menuAnchor}
-        >
-          <OverflowMenuIcon />
-        </HeaderButton>
-        <Menu
-          id="more-menu"
-          anchorEl={menuAnchor.current}
-          keepMounted
-          open={menuOpen}
-          onClose={handleClose}
-        >
-          <MenuItem onClick={() => null}>Placeholder</MenuItem>
-        </Menu>
       </Box>
-
       <UserSummary user={otherUser}>
-        <HostingStatus hostingStatus={otherUser?.hostingStatus} />
+        {hostRequest && (
+          <div className={classes.requestedDatesWrapper}>
+            <Typography
+              component="p"
+              variant="h3"
```

Comment:
I think this is good, you want the header styling but it isn't a header. So why not.

Reply:
No moreso the (2 nights) bit. That has the same component/variant combination, but with fontWeight: initial, just because body1 and body2 don't change fontSize dynamically (whereas h3 does).